modules/programs/alot.nix

```diff
@@ -164,7 +192,7 @@ in
 
     xdg.configFile."alot/config".text = configFile;
 
-    xdg.configFile."alot/hooks.py".text =
+    xdg.configFile."alot/hooks.py".text = optionalString (cfg.hooks != "")
```

Comment:
Nice idea. But I think this would still write the file, just leave it empty. Perhaps something like

```
xdg.configFile."alot/hooks.py" = mkIf (cfg.hooks != "") {
  text = ''
    …
  '';
};
```